wp-content/themes/borderzine/less/inc/header.less

```diff
@@ -17,4 +17,41 @@
 #main-nav.navbar {
 	border-top: none;
 	border-bottom: 2px solid @orange;
-}
\ No newline at end of file
+}
+.subscribe-btn {
+    float: left;
+    margin: 5px 0 0 10px;
+    background: @white;
+    font-size: 14px;
+    line-height: 2;
+    a {
+        padding: 24px 7px;
+        color: @orange;
+        &:hover {
+            text-decoration: none;
+            color: @orange;
+        }
+    }
+    &:hover {
+        background: darken(@white, 5%);
+    }
+}
+.subscribe {
+    .subscribe-link {
+        span {
+            padding: 6px 8px;
```

Comment:
should be 4px 8px to match the donate button.

Reply:
Whoops. Good catch.

Reply:
Fixed in https://github.com/INN/umbrella-borderzine/pull/84/commits/9835e2a610507b1e052cdb24c4cd4bbaf12ceb89